test/src/034-cachecleanup/main

```diff
@@ -2,21 +2,132 @@
 
 cvmfs_test_name="Cache cleanup"
 
```

Comment:
Can you add a short "docstring" style comments to the test, summarizing what it is intending to test?